ui/pages/confirmations/send/send-content/add-recipient/add-recipient.container.test.js

```diff
@@ -48,9 +52,13 @@ describe('add-recipient container', () => {
     it('should map the correct properties to props', () => {
       expect(mapStateToProps('mockState')).toStrictEqual({
         addressBook: [{ name: 'mockAddressBook:mockState' }],
-        addressBookEntryName: undefined,
         contacts: [{ name: 'mockAddressBook:mockState' }],
-        domainResolution: 'mockSendDomainResolution:mockState',
+        domainResolutions: [
+          {
+            resolvedAddress: 'mockSendDomainResolutions:mockState',
+            protocol: 'Ethereum Name Service',
```

Comment:
Missing `addressBookEntryName` and  `resolvingSnap`

Reply:
Resolving this because both entries can be `undefined` and in this scenario it makes sense to leave out.